x-pack/plugins/lens/public/visualizations/metric/dimension_editor.tsx

```diff
@@ -299,17 +263,176 @@ function PrimaryMetricEditor(props: SubProps) {
 
   const togglePalette = () => setIsPaletteOpen(!isPaletteOpen);
 
+  const supportingVisLabel = i18n.translate('xpack.lens.metric.supportingVis.label', {
+    defaultMessage: 'Supporting visualization',
+  });
+
+  const hasDefaultTimeField = props.datasource?.hasDefaultTimeField();
+  const metricHasReducedTimeRange = Boolean(
+    state.metricAccessor &&
+      props.datasource?.getOperationForColumnId(state.metricAccessor)?.hasReducedTimeRange
+  );
+  const secondaryMetricHasReducedTimeRange = Boolean(
+    state.secondaryMetricAccessor &&
+      props.datasource?.getOperationForColumnId(state.secondaryMetricAccessor)?.hasReducedTimeRange
+  );
+
+  const supportingVisHelpTexts: string[] = [];
+
+  const supportsTrendline =
+    hasDefaultTimeField && !metricHasReducedTimeRange && !secondaryMetricHasReducedTimeRange;
+
+  if (!supportsTrendline) {
+    supportingVisHelpTexts.push(
+      !hasDefaultTimeField
+        ? i18n.translate('xpack.lens.metric.supportingVis.needDefaultTimeField', {
+            defaultMessage: 'Use a data view with a default time field to enable trend lines.',
```

Comment:
Suggest changing this text to: `Line visualizations require use of a data view with a default time field.`

Reply:
For some reason, I thought we we generally aim to give the user direction. That's why I used the imperative voice.

Reply:
Ultimately, I'll defer to @KOTungseth for all of my wording suggestions here. However, my reasoning for suggesting this direction (and moving away from imperative) was to put the name of the visualization being referenced first in the sentence for easier scanning (since there exists the possibility of multiple help text sentences for across multiple visualization types (bar, line).

Reply:
Okay, I'll implement your suggestions in https://github.com/elastic/kibana/pull/143781 and @KOTungseth can chime in there next week if she has time 👍

Reply:
How about:

`To create line charts, you must use a data view with a default time field.`

Reply:
I worry that "To create line charts" implies greater flexibility than we offer with the metric trend lines (on/off switch). I wonder if even getting away from calling them trend lines is a good idea (getting back to Michael's original feedback on the button itself).

WDYT? I ultimately defer to you 👍